external/vcm/vcm/cubedsphere/grid_metadata.py

```diff
@@ -0,0 +1,24 @@
+import dataclasses
+
+
+@dataclasses.dataclass
+class GridMetadata:
+    x: str
+    y: str
+    x_interface: str
+    y_interface: str
+    tile: str = "tile"
+    lon: str = "lon"
+    lonb: str = "lonb"
+    lat: str = "lat"
+    latb: str = "latb"
```

Comment:
Maybe just give everything defaults? Not really clear what distinguishes the dimension names from the lat/lon variable names. If anything, I would say the former are less likely to change than the latter.

Reply:
Sure will add.